examples/http-ts/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import * as http from 'http';
+import * as store from '@leapfrogtechnology/async-store';
+import AsyncStoreAdapter from '@leapfrogtechnology/async-store/dist/AsyncStoreAdapter';
+
+import router from './router';
+import * as logger from './logger';
+import { BASE_URL, PORT } from './config';
```

Comment:
No need extra file for these two constants as this we'd like to simplify and shorten this example as much as possible.